src/github_review_prompts/main.py

```
@@ -580,6 +596,206 @@ def colorize(text: str, color_code: str) -> str:
+        # インラインコメント（line番号が存在）のみ対象
+        if comment.get("line") is None:
+            return False
+
+        # 時刻条件は複雑なため、現在は適用しない
+        # TODO: 将来的にはコメント対応の修正コミット特定ロジックを実装
+        # if last_commit_time:
+        #     comment_time = comment.get("created_at")
+        #     if comment_time and comment_time > last_commit_time:
+        #         return False
+
+        return True
```

Ruff (0.12.2)

Return the negated condition directly

(SIM103)